llm-complete-guide/configs/production/synthetic.yaml

```
@@ -0,0 +1,39 @@
+# environment configuration
+settings:
+  docker:
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```

```suggestion
      - ratelimit
```

Again, no langchain needed

---

you might need to add `openai`, though...